scripts/inference/hf_generate.py

```diff
@@ -20,7 +22,7 @@ def get_dtype(dtype):
         return torch.bfloat16
     else:
         raise NotImplementedError(
-            f'dtype {dtype} is not supported. '
+            f'dtype {dtype} is not supported. ' +\
```

Comment:
Is this a new syntax that pyright and lint are enforcing? Just curious, I haven't used it before

Reply:
It's not being enforced, but my IDE hates implicit string concatenation and it has convinced me of the same.

I always see some annoying warning reminding me why the `+\` explicit string concatenation is safer/better, and it honestly just makes the code so much less ugly and distracting when the warning highlights go away.